e2e/webdriver/chrome.js

```diff
@@ -0,0 +1,60 @@
+const {Builder} = require('selenium-webdriver')
+const chrome = require('selenium-webdriver/chrome')
+
+/**
+ * A wrapper around a {@code WebDriver} instance exposing Chrome-specific functionality
+ */
+class ChromeDriver {
+  static async build ({extensionPath}) {
+    const options = new chrome.Options()
+      .addArguments([
+        `load-extension=${extensionPath}`,
+      ])
+      .setChromeBinaryPath('/Applications/Google Chrome Canary.app/Contents/MacOS/Google Chrome Canary')
```

Comment:
This is an OS, and possibly individual system, dependent path

Reply:
My bad, good catch

Reply:
Fixed